topics/transcriptomics/tutorials/scrna-raceid/tutorial.md

```diff
@@ -0,0 +1,712 @@
+> > `{2, {8 {12 {10}}, 11, 4, 9, 3 {5, 1 { 6 } } } }` (in JSON format)
+> > 
+> >
+> {: .solution}
+>
+{: .question}
+
+
+![Link scores]({{site.baseurl}}{% link topics/transcriptomics/images/raceid_stemid_linkscores.png %} "StemID Link scores between branches.")
+
+The top chart shows the number of links above a threshold that each cluster exhibits to another. The more links a cluster has, the more evidence that the cluster describes a progenitor cell type that gives rise to other more mature types. The middle chart describes the "Delta-Entropy" which measures the level of variability within a cluster, where clusters with more variability are less likely to be mature cell types due to the sheer "noise" that they exhibit that is to be expected of a cell type that could potentially give rise to other types. The bottom chart is simply the top chart multiplied by the middle chart, which adds both pieces of evidence together to yield the link score.
+
+> ### {% icon question %} Questions
+>
+> With this new information, which cluster is now most likely to be the sole progenitor?
+>
+> > ### {% icon solution %} Solution
+> >
+> > `c3` has the most number of links, and the most entropy that one would expect a multipotent progenitor cell type to exhibit, and therefore must be the root of the lineage tree, *despite* having fewer links than `c2`.
+> > `{3 {5, 1 {6}, 2 {8 {12 {10}}, 11, 4}}}` (in JSON format)
+> >
+> {: .solution}
+>
+{: .question}
+
+<!--
+![Other link scores]({{site.baseurl}}{% link topics/transcriptomics/images/raceid_stemid_other.png %} "Other link scores between branches.")
+-->
+
+In a similar vein to how clusters were explored individually in RaceID, we can also explore individual branches of the lineage tree to see how some genes are up or down regulated.
+
+## Specific Trajectory Lineage Analysis (StemID)
+
+Here we will explore one branching point of interest; `c3` giving rise to `c1` and `c5`. Will we be able to find an up or down regulation of genes between these two branches?
+
+
+> ### {% icon hands_on %} Hands-on: Comparing trajectory paths 3 to 1, and 3 to 5
+>
+> 1. **Lineage Branch Analysis using StemID** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Input RDS"*: `outrdat` (output of **Lineage computation using StemID** {% icon tool %})
+>    - In *"StemID Branch Link Examine"*:
+>        - *"Perform StemID?"*: `Yes`
+>            - *"Cluster Number"*: `1`
+>            - *"Trajectory Path i, j, k"*: `1,3,5`
+>            - *"Use Defaults?"*: `Yes`
+>    - In *"FateID Branch Link Examine"*:
+>        - *"Perform FateID?"*: `No`
+>
+{: .hands_on}
+
+![Heatmap Cells]({{site.baseurl}}{% link topics/transcriptomics/images/raceid_stemid_cells.png %} "StemID Heatmap of Cluster 1 cell projections.")
+
+The vertical names along the heatmap are the cells from `c1` being projected on to the two intercluster links (3 to 1, 3 to 5) with scores for each cluster being shown for that cell. 
```

Comment:
I do not understand what is represented in the heatmap and how it is related to the trajectory Lineage Analysis